lib/__tests__/harvest.test.ts

```diff
@@ -102,7 +106,7 @@ describe('Integration tests', () => {
     expect(data).toHaveProperty('queryID')
     expect(requestClick.queryID).toBe(data.queryID)
 
-    expect(requestClick.objectID).toBe('81')
+    expect(requestClick).toHaveProperty('objectID')
```

Comment:
I've also changed this tests because I thought it made this tests to tightly coupled to one particular index.